crux-test/test/crux/kv/index_store_test.clj

```diff
@@ -274,3 +274,17 @@
       (t/is (= {:crux.db/id {:doc-count 3675, :values 3554, :eids 3554}
                 :sub-idx {:doc-count 3675, :values 50, :eids 3554}}
                (->stats *index-store*))))))
+
+(t/deftest test-entity
```

Comment:
This namespace might be a good place for a slowdown factor-style test, sanity checking the indexing and reconstruction cost of the new approach?